src/App.tsx

```diff
@@ -15,49 +18,57 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+type SortOptions = {
+  sortField: SortField;
+  isReversed: boolean;
+};
+
+function getPreparedGood(goods: string[], options: SortOptions) {
+  const { sortField, isReversed } = options;
+
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortField.Alphabet:
+          return good1.localeCompare(good2);
+
+        case SortField.Length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App: FC = () => {
+  const [sortField, setSortField] = useState<SortField>(SortField.Default);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
You don't have to specify it. TypeScript understands it itself.